records/scripts/pullProd.ts

```diff
@@ -0,0 +1,54 @@
+const token = Deno.env.get("DATABASE_AUTH_TOKEN");
+const sitename = Deno.env.get("DECO_SITE_NAME");
+
+const error = "Could not pull production database";
+
+async function run() {
+  if (!token || !sitename) {
+    console.error(
+      "DATABASE_AUTH_TOKEN and DECO_SITE_NAME environment variables must be set in order to pull the production database.",
+    );
+    throw error;
+  }
+
+  const productionDumpUrl = `https://${sitename}-decocx.turso.io/dump`;
```

Comment:
[Shouldn't you get this from the app's config?https://www.getlago.com/](https://www.getlago.com/)

Reply:
This runs outside the deco flow. Do you to suggest that should be inside the deco flow?